cpt/test/unit/generators_test.py

```diff
@@ -337,7 +337,7 @@ def test_visual_build_generator(self):
                                    shared_option_name="libpng:shared",
                                    dll_with_static_runtime=False,
                                    vs10_x86_64_enabled=False,
-                                   build_types=["Debug", "Release"])
+                                   build_types=["Debug", "Release", "RelWithDebInfo", "MinSizeRel"])
```

Comment:
Same here (I would leave all of them as they were, and would introduce a check of the size of the builds)